.dockerignore

```diff
@@ -1,6 +1,4 @@
 .git
 .gitignore
-node_modules/
```

Comment:
why are we removing this?

Reply:
without this, i was getting the typegraphql error; if it works without this, not needed